static/app/components/replays/diff/replaySliderDiff.tsx

```diff
@@ -19,15 +19,22 @@ import {useResizableDrawer} from 'sentry/utils/useResizableDrawer';
 
 interface Props {
   leftOffsetMs: number;
+  minHeight: `${number}px` | undefined;
```

Comment:
This is not optional, (ie: not `minHeight?:`) because i think it should be set each time.  When this is set to `undefined` its possible for it to look like this and be frusterating:
<img width="896" alt="SCR-20240909-niqw" src="https://github.com/user-attachments/assets/480f5aa9-f3db-42f2-a962-bdef556f7087">

Reply:
Why allow `undefined` then?

Reply:
if the parent node is setup properly, then undefined works to give it an unconstrained height growing and shinking

Reply:
i guess, i could see what `0px` looks like and update the defaults